### PR TITLE
Orientation selector

### PR DESCRIPTION
**Pull request by schaumb**

### CODE REVIEW

**Comment by simzer** [comment]
reason for some test fail: up to this point, orientation selection run after all individual config parameter change. This caused a dependence of the final orientation on the order of the x, y channel setting. This was an unwanted behavior.

**Comment by schaumb** [comment]
If I only sort X and Y, only 27 tc fails.

File: src/chart/animator/planner.cpp [hunk 1]
```diff
@@ -336,7 +336,7 @@ bool Planner::verticalBeforeHorizontal() const
 	const auto &srcOpt = source->getOptions();
 	const auto &trgOpt = target->getOptions();
 
-	if (srcOpt->horizontal != trgOpt->horizontal
+	if (*srcOpt->orientation.get() != *trgOpt->orientation.get()
```
Comment (simzer) [inline src/chart/animator/planner.cpp:339]: Could be isHorizontal() as well.

**Review by simzer — APPROVED** [review]

**Review by simzer — CHANGES_REQUESTED** [review]
Please add a note to the changelog

**Review by simzer — APPROVED** [review]

**Pull request merged**